app/controllers/kudos_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class KudosController < ApplicationController
+  before_action :set_kudos, only: %i[show edit update destroy]
+  before_action :authenticate_employee!, except: %i[index show]
+  before_action :correct_employee, only: %i[edit update destroy]
```

Comment:
there is no definition of `:correct_employee` - what it was supposed to do anyway?

Reply:
you are right it is unnecessary